src/Amazon.Extensions.S3.Encryption.csproj

```diff
@@ -15,43 +15,27 @@
         <RepositoryUrl>https://github.com/aws/amazon-s3-encryption-client-dotnet/</RepositoryUrl>
         <Company>Amazon Web Services</Company>
-        <AssemblyVersion>2.1.2</AssemblyVersion>
-        <FileVersion>2.1.2</FileVersion>
-
         <SignAssembly>true</SignAssembly>
         <AssemblyOriginatorKeyFile>..\public.snk</AssemblyOriginatorKeyFile>
-
-        <!-- workaround per https://github.com/Microsoft/msbuild/issues/1333 -->
-        <FrameworkPathOverride Condition="'$(TargetFramework)' == 'net35'">C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v3.5\Profile\Client</FrameworkPathOverride>
-
-        <!-- workaround per https://github.com/dotnet/msbuild/issues/5985 -->
-        <AutomaticallyUseReferenceAssemblyPackages Condition=" '$(TargetFramework)' == 'net35' ">false</AutomaticallyUseReferenceAssemblyPackages>
+        <PackageReadmeFile>README.md</PackageReadmeFile>
+    </PropertyGroup>
+    
+    <PropertyGroup Condition="'$(TargetFramework)' == 'net472'">
+        <AssemblyVersion>3.0.0</AssemblyVersion>
     </PropertyGroup>
 
     <ItemGroup>
         <None Include="../LICENSE" Pack="true" PackagePath="" />
         <None Include="../icon.png" Pack="true" PackagePath="" />
+        <None Include="../README.md" Pack="true" PackagePath="" />
     </ItemGroup>
 
-    <PropertyGroup>
-        <DefineConstants Condition="'$(TargetFramework)' == 'net35'">$(DefineConstants);BCL;BCL35;AWS_APM_API</DefineConstants>
-        <DefineConstants Condition="'$(TargetFramework)' == 'net45'">$(DefineConstants);BCL;BCL45;AWS_ASYNC_API</DefineConstants>
-        <DefineConstants Condition="'$(TargetFramework)' == 'netstandard2.0'">$(DefineConstants);NETSTANDARD;AWS_ASYNC_API</DefineConstants>
-        <DefineConstants Condition="'$(TargetFramework)' == 'netcoreapp3.1'">$(DefineConstants);NETSTANDARD;AWS_ASYNC_API</DefineConstants>
-    </PropertyGroup>
-
     <ItemGroup>
-        <PackageReference Include="AWSSDK.Core" Version="3.7.303.14" />
-        <PackageReference Include="AWSSDK.S3" Version="3.7.307.15" />
-        <PackageReference Include="AWSSDK.KeyManagementService" Version="3.7.301.15" />
-    </ItemGroup>
-
-    <ItemGroup Condition="'$(TargetFramework)' == 'net35'">
-        <PackageReference Include="BouncyCastle" Version="1.8.9" />
-    </ItemGroup>
-
-    <ItemGroup Condition="'$(TargetFramework)' == 'net45' Or '$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == 'netcoreapp3.1'">
-        <PackageReference Include="Portable.BouncyCastle" Version="1.8.10" />
+        <PackageReference Include="AWSSDK.Core" Version="4.0.0-preview.2" />
+        <PackageReference Include="AWSSDK.S3" Version="4.0.0-preview.2" />
+        <PackageReference Include="AWSSDK.KeyManagementService" Version="4.0.0-preview.2" />
+        <PackageReference Include="BouncyCastle.Cryptography" Version="2.4.0" />
+        <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
```

Comment:
I think this dependency should only be here for `net472` and `netstandard2.0` targets. Can you also add a TODO to remove this once we fix the nuspec in core?

Reply:
Done